src/llv8-constants.cc

```diff
@@ -37,57 +37,70 @@ void Module::Assign(SBTarget target, Common* common) {
   common_ = common;
 }
 
+uint64_t BufToUint64(const uint8_t* buf, uint32_t size) {
+  uint64_t res = 0;
```

Comment:
It should be possible to support big endian system by using `target.GetByteOrder()`, but I am not sure if we do actually support big endian system at the moment? Also I don't really have a box to test for that..

Reply:
Ah nevermind, I think I can just `memcpy` to work around the endianness..